coderd/database/databasefake/databasefake.go

```diff
@@ -923,6 +977,13 @@ func (q *fakeQuerier) GetParameterValueByScopeAndName(_ context.Context, arg dat
 	return database.ParameterValue{}, sql.ErrNoRows
 }
 
+func (q *fakeQuerier) GetTemplates(_ context.Context) ([]database.Template, error) {
+	q.mutex.RLock()
+	defer q.mutex.RUnlock()
+
+	return q.templates, nil
+}
```

Comment:
Should this copy `q.templates`? We can't enforce concurrency on it otherwise.

Reply:
Will fix